### ISSUE TITLE
Markup control's Command property bound using Static Command Service does not work

### ISSUE THREAD

**Issue by vstribrny** [riganti/dotvvm#643]
I have a Command property in my Markup Control and bind it in root View Model using Static Command Service.  When command is about to be invoked, it throws with:

> System.InvalidOperationExceptionCould not implicitly convert expression of type System.Void to System.Object.

Please see [GridViewAndMarkupControlCommand](https://github.com/vstribrny/DotVVM-issues-repro/tree/master/src/GridViewAndMarkupControlCommand) to reproduce the problem.

After some time of investigation, I was able to somehow hack it. However, it's not documented/recommended, is it? See below:

```
// DOES NOT WORK
[AllowStaticCommand]
public void OnChanged(Item item, RootItem rootItem) 
{
    rootItem.MyText = item.Number.ToString();
}
```
```
// WORKS
[AllowStaticCommand]
public System.Delegate OnChangedHacked(Item item, RootItem rootItem) 
{
    rootItem.MyText = item.Number.ToString();
    return new DotVVM.Framework.Binding.Expressions.Command(() => Task.CompletedTask);
}
```

**Comment by exyi** [comment]
It seems that there are multiple glitches nested in themselves...

First, there is a "by design" glitch which basically makes this usage of `staticCommand` binding impossible. When you are trying to invoke the command in a control, you are doing a full postback that will build the entire control tree, find the corresponding control and invoke the function **on it**. So you can do the same thing with `command` binding, it will just have less restrictions.

The second issue is that the implicit conversions of the bindable functions are screwed up a bit in this case. I'll have a look at it, but it will not help you solve the problem :(

**Comment by exyi** [comment]
The exception should be fixed at 51abc71, I'm closing the issue. Thank you for reporting it.

Note that the combination of `staticCommand` and `controlCommand` will still not work very well, it will essentially invoke the `staticCommand` using standard postback mechanism.
